src/pages/docs/[[...filePath]].tsx

```diff
@@ -0,0 +1,173 @@
+import renderToString from "next-mdx-remote/render-to-string";
+import hydrate from "next-mdx-remote/hydrate";
+import path from "path";
+import { getMdxPaths } from "@vulcanjs/mdx";
```

Comment:
It triggers an error in the pipeline but ignore it (this is because we haven't released corresponding Vulcan NPm version yet, that's normal)